site2/website/pages/en/index.js

```diff
@@ -168,30 +168,7 @@ const UsersBlock = props => (
     id={props.id}
     background={props.background}>
 
-    <p align="center"><small style={{color: 'black', fontSize: '1.7rem'}}>Used by companies such as</small></p>
-    <div class="logo-wrapper">
-      {
-        featuredUsers.map(
-            c => (
-                (() => {
-                  if (c.hasOwnProperty('logo_white')) {
-                    return <div className="logo-box-background-for-white">
-                      <a href={c.url} title={c.name} target="_blank">
-                        <img src={c.logo} alt={c.name} className={c.logo.endsWith('.svg') ? 'logo-svg' : ''}/>
-                      </a>
-                    </div>
-                  } else {
-                    return <div className="logo-box">
-                      <a href={c.url} title={c.name} target="_blank">
-                        <img src={c.logo} alt={c.name} className={c.logo.endsWith('.svg') ? 'logo-svg' : ''}/>
-                      </a>
-                    </div>
-                  }
-                })()
-            )
-        )}
-    </div>
-    <p align="center"><small style={{color: 'black', fontSize: '1.7rem'}}><a href="/powered-by">... and many more</a></small></p>
+    <p align="center"><small style={{color: 'black', fontSize: '1.7rem'}}><a href="/powered-by">Used by companies such as ...</a></small></p>
```

Comment:
@dave2wave @eolivelli The text will now look incomplete with the dots

Reply:
Sure let's wordsmith:
c/Used by companies such as .../Click to see some of the Companies Powered by Pulsar/